doc/whats_new/v1.4.rst

```diff
@@ -36,6 +36,11 @@ The following models now support metadata routing in one or more or their
 methods. Refer to the :ref:`Metadata Routing User Guide <metadata_routing>` for
 more details.
 
+- |API|:func:`~utils.metadata_routing.process_routing` now has a different
+  signature. The first two (the object and the method) are positional only,
+  and all metadata are passed as keyword arguments. :pr:`26909` by `Adrin
+  Jalali`_.
```

Comment:
if you want to include this here, we should change the description of the section which is:

```
The following models now support metadata routing in one or more or their
methods. Refer to the :ref:`Metadata Routing User Guide <metadata_routing>` for
more details.
```

I intentionally had left these out to keep this section about models. I don't mind moving these entries here, but we then need to fix the section description.

Reply:
OK I see, I reverted this 2 changes.